metrics/grafana/tikv_details.dashboard.py

```diff
@@ -2502,6 +2534,7 @@ def RaftProcess() -> RowPanel:
                             0.99,
                             "tikv_raft_apply_ahead_of_persist",
                             by_labels=["instance"],
+                            is_optional_quantile=True,
                         ),
                         legend_format="{{instance}}-99%",
```

Comment:
```suggestion
                        legend_format="{{instance}}-$optional_quantile",
```
Include in the legend as well?

Reply:
Thx for reminding.

Reply:
And the title also includes the `option`.